modules/life-cycle-mandatory-upgrades.adoc

```diff
@@ -10,7 +10,4 @@ impacts the security or stability of the cluster, the customer must upgrade to t
 patch release within two link:https://access.redhat.com/articles/2623321[business days].
 
 In extreme circumstances and based on Red Hat's assessment of the CVE criticality to the
-environment, if the upgrade to the next supported patch release has not been performed within
-two link:https://access.redhat.com/articles/2623321[business days] of notification, the cluster
-will be automatically updated to the latest patch release to mitigate potential security breach
-or instability.
\ No newline at end of file
+environment, Red Hat will notify customers that they have two link:https://access.redhat.com/articles/2623321[business days] to schedule or manually update their cluster to the latest, secure patch release. In the case that an update has not been performed, Red Hat will automatically update the cluster to the latest, secure patch release to mitigate potential security breach or instability. Red Hat may, at its own discretion, temporarily delay an automated update if requested by a customer through a link:https://access.redhat.com/support[support case].
```

Comment:
s/potential/a potential

Reply:
I am no expert here, but this section now sounds almost contradictory. 

```
In the event that a Critical or Important CVE, or other bug identified by Red Hat, significantly impacts the security or stability of the cluster, the customer must upgrade to the next supported patch release within two business days.

In extreme circumstances and based on Red Hat’s assessment of the CVE criticality to the environment, Red Hat will notify customers that they have two business days to schedule or manually update their cluster to the latest, secure patch release.
```
The customer must **always** upgrade in 2 days if there is a bug that impacts the security or stability of the cluster, or only when Red Hat tells users to upgrade in 2 days? Or, am I conflating things?

Reply:
It's always two business days, but then in extreme circumstances, after the two business days have passed, if the customer doesn't update, RH reserves the right to force the update. That's how I interpret it.

Reply:
@wgordon17 Can you take a look at my concern here? Your changes on their own make perfect sense. However, when read with the first paragraph, it comes off as confusing to me.

Reply:
@wgordon17 Can you look at @mburke5678 's question above and weigh in? Thank you!

Reply:
Sorry, for the delay. So the implications are

- First paragraph "the customer must upgrade to the next supported patch release within two business days" states RH's expectations of the customer: "Upgrade your cluster in 2 days"

- Second paragraph "In extreme circumstances and based on Red Hat's assessment of the CVE criticality ... Red Hat will notify customers that they have two business days to ... update their cluster ... [or] Red Hat will automatically update the cluster" states what the customer's expectations should be of RH.

It's entirely possible that there could be some confusion between the 2 paragraphs

Reply:
@wgordon17 Thank you!
@mburke5678 Tagging you so you can read Will's response. Thoughts?

Reply:
@wgordon17 @eohartman 

In both scenarios, the CU has two days to upgrade. However in extreme circumstances, RH reaches out to the CU. In that case, RH will do the upgrade if the CU doesn't do it on time. Is that it?  
It wasn't clear that there were two scenarios.
If so, what about: "However, in extreme circumstances..." to emphasize that this describes two scenarios. 

BTW, what happens in the first scenario if the user doesn't upgrade in 2 days?

Reply:
This pull request has been open for two weeks - I'd like to wrap this up in by EOD Tuesday, Feb. 14, so I can mark it as resolved. @mburke5678 @wgordon17 is this possible?

Reply:
Apologies for the delay, everything here looks good 👍 Thanks everyone 🙌